src/mapwindow.jl

```diff
@@ -190,14 +193,14 @@ function _intersectionindices(full::AbstractUnitRange, r::AbstractRange)
     else
         ret = _indexof(r,first(r_sub)):_indexof(r,last(r_sub))
     end
-    @assert intersect(full, r) == r[ret]
+    @assert r_sub == r[ret] || isempty(r_sub) && isempty(ret)
     ret
 end
 
 function _indexof(r::AbstractRange, x)
     T = eltype(r)
```

Comment:
Interestingly, this should really be `eltype(axes(r, 1))`, but that's always `Int`.